app/models/classes/format_url.rb

```diff
@@ -0,0 +1,18 @@
+class Classes::FormatUrl
```

Comment:
Sorry Dan do you mean in the helpers folder and then create a folder called post_helpers?